Readme.md

```diff
@@ -10,8 +10,20 @@ a cookie for tracking sessions.
 + [fastify-caching](https://www.npmjs.com/package/fastify-caching): used to
 store the session data via the `fastify.cache` decorator.
 
+
+**Installation:**
+
+```
+npm install fastify-server-session fastify-cookie fastify-caching
+```
```

Comment:
Do we really need this? I think people understand how to manage packages at this point.

Reply:
I personally find it's a good pattern as there is other dependencies. Also some repositories have Github repository names that does not equal the npm package name. Avoid potential confusion in a sense